src/utils/bootstrap/bootstrap.service.ts

```diff
@@ -25,13 +28,52 @@ export class BootstrapService {
     try {
       console.info('Bootstrapping Ecoverse...');
       await this.ensureEcoverseSingleton();
-      await this.ensureAdminRole();
+      await this.bootstrapProfiles();
       await this.validateAccountManagementSetup();
     } catch (error) {
       console.log(error);
     }
   }
 
+  async bootstrapProfiles() {
+    const bootstrapFilePath = this.configService.get<IServiceConfig>('service')
+      ?.authorisationBootstrapPath as string;
+
+    const bootstrapData = await import(bootstrapFilePath);
+    await this.createGroupProfiles(
+      RestrictedGroupNames.Admins,
+      await bootstrapData[RestrictedGroupNames.Admins]
```

Comment:
this looks strange - awaiting on navigating to data that is already loaded? Probably a good idea to have a defensive check that each expected group is present.

Reply:
Fixed in d6f43ad5ea6b78b4b79d6889962971a5eae009eb